app/resources/shader/Face.fragsh

```diff
@@ -48,7 +49,7 @@ void main() {
 
     // Assume alpha masked or opaque.
     // TODO: Make this optional if we gain support for translucent textures
-    if (gl_FragColor.a < 0.5) {
+    if (!DisableBlend && gl_FragColor.a < 0.5) {
         discard;
```

Comment:
What the comment is referring to is that the "discard" breaks alpha translucency since it assumes we're either doing Opaque or Masked rendering. We should also set `DisableBlend` to true when doing blending to prevent the `discard` from running, and maybe rename it for clarity to "DisableMasked"

In the longer run we should use a preprocessor to compile separate shaders for masked and opaque or translucent rendering; having "discard" in the program disables some optimizations (early-z?)

Reply:
Renamed to `EnableBlend`.